CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.
 - Use 0.0.0-dev product images for testing ([#441]).
 - Use testing-tools 0.2.0 ([#441]).
 - Added kuttl test suites ([#455])
+- Set resource limits on containers by default ([#456])
```

Comment:
I would suggest to rename the entry above to `- Set explicit resources on all containers ([#453], [#456]).`

Reply:
changed